src/services/walktroughServices.js

```diff
@@ -0,0 +1,109 @@
+import { create, watch, currentUser, OpenShiftWatchEvents } from './openshiftServices';
+import {
+  cleanUsername,
+  findOpenshiftResource,
+  buildServiceInstanceResourceObj,
+  findOrCreateOpenshiftResource
+} from './middlewareServices';
+import * as osKinds from '../common/openshiftKinds';
+import { middlewareTypes } from '../redux/constants';
+import { FULFILLED_ACTION } from '../redux/helpers';
+import { DefaultServiceInstanceTransform } from '../common/serviceInstanceHelpers';
+
+/**
+ * Node messaging app transform, adds enmass credentials
+ */
+class NodeMessagingAppEnmassSiTransform {
+  isTransformable(siInfo) {
+    return siInfo.name === 'nodejs-messaging-work-queue-frontend';
+  }
+
+  transform(siInfo) {
+    const enmassCreds = {
+      MESSAGING_SERVICE_USER: 'username',
+      MESSAGING_SERVICE_PASSWORD: 'password'
+    };
+    const defaultTransform = new DefaultServiceInstanceTransform().transform(siInfo);
+    defaultTransform.spec.parameters = enmassCreds;
+    return defaultTransform;
+  }
+}
+
+const walkthroughs = {
+  one: {
+    namespaceSuffix: 'walkthrough-one',
+    services: ['spring-boot-rest-http-crud', 'nodejs-messaging-work-queue-frontend'],
+    transforms: [new DefaultServiceInstanceTransform()]
+  },
+  oneA: {
+    namespaceSuffix: 'walkthrough-one-a',
+    services: ['spring-boot-rest-http-crud', 'nodejs-messaging-work-queue-frontend'],
+    transforms: [new NodeMessagingAppEnmassSiTransform(), new DefaultServiceInstanceTransform()]
```

Comment:
Is there a reason why we're including `DefaultServiceInstanceTransform` here?

Reply:
hey, it transforms the spboot app, at the moment only node app needs parameters
I was going to include defaulttransform as a fallback, so we dont have to always add it as last, but decided against it as it might cause confusion.